FirebaseDynamicLinks/CHANGELOG.md

```diff
@@ -4,6 +4,8 @@
 - [fixed] Reduce frequency of iOS14 pasteboard notifications by only reading from it when it contains URL(s). (#5905)
 - [changed] Functionally neutral updated import references for dependencies. (#5824)
 
+Refer to [README.md](https://github.com/firebase/firebase-ios-sdk/blob/master/FirebaseDynamicLinks/README.md) for more details about this release.
```

Comment:
nit: This should be "Refer to the README" instead of "Refer to README"

Reply:
added "the"